resources/lang/fr/general.json

```diff
@@ -0,0 +1,5 @@
+{
+    "all": "Tous",
+    "date": "Date",
+    "empty-logs": "Lalistedeslogsestvide!"
```

Comment:
This is one example of removed spaces compared to the [original](https://github.com/ARCANEDEV/LogViewer/blob/master/resources/lang/fr/general.php#L6)